content/articles/geospatial_standards/index.Rmd

```diff
@@ -0,0 +1,92 @@
+- Although the previous version of the GeoJSON standard -- GeoJSON 2008 -- is still a lot in use, it is [obsoleted](http://geojson.org/geojson-spec.html) and a new version **[RFC7946](https://tools.ietf.org/html/rfc7946)** is establishing.
+  - This version is strict about the coordinate reference system (CRS) -- it is always [WGS84](https://epsg.io/4326) -- and it also differs on a few other aspects (such as the recommendation for applications [not to inflate](https://tools.ietf.org/html/rfc7946#section-11.2) decimal coordinate precision).
+  - RFC7946 solves the problem that quite a few libraries -- including GDAL -- simply assumed WGS84 in GeoJSON 2008 (without checking or transforming), even though WGS84 was not a requirement of GeoJSON 2008 (it did support an explicit _crs_ declaration).
+This resulted in inconveniences (cf. [this post](https://github.com/r-spatial/sf/issues/344#issue-229118527) in the `sf`-package).
+  - A [specific section](https://gdal.org/drivers/vector/geojson.html#rfc-7946-write-support) in the documentation of GDAL's GeoJSON driver gives a summary of the differences between both GeoJSON versions.
+- While GDAL by default still follows the GeoJSON 2008 format ^[
+Though GeoJSON 2008 is obsoleted, the now recommended RFC7946 standard is still officially in a _proposal_ stage.
+That is probably the reason why GDAL does not yet default to RFC7946.
+A somehow confusing stage, it seems.
+], 
+RFC7946 is supported by the option `RFC7946=YES`.
+Here, on-the-fly reprojection to WGS84 will happen automatically.
+It applies 7 decimal places for coordinates, i.e. approximately 1 cm.
+Given the advantages, _**we advise to explicitly use RFC7946**_.
+Several functions in R allow the user to provide options that are passed to GDAL, so we can ask to deliver RFC7946 (see the [tutorial](../../tutorials/spatial_standards_vector/)).
+- In order to keep it manageable (text file size, usage in versioning systems) it can be wise to use GeoJSON for more simple cases (points and rather simple lines and polygons), and use the binary GeoPackage format for larger (more complex) cases.
+
+## The GeoTIFF file format
+
+- [GeoTIFF](https://en.wikipedia.org/wiki/GeoTIFF) is the preferred single-file open standard for **raster** data.
+It adheres to the open [TIFF](https://en.wikipedia.org/wiki/TIFF) specification; hence it is a TIFF image file (`filename.tif`).
+It [uses](http://docs.opengeospatial.org/is/19-008r4/19-008r4.html#_geotiff_file_structure_and_geotiff_crs_and_models_principles_informative) a small set of reserved TIFF tags to store information about CRS, extent and resolution of the raster.
+- A GeoTIFF file can contain _one_ or _multiple_ rasters with the same CRS, extent and resolution.
+- The GeoTIFF standard is [maintained](https://www.opengeospatial.org/standards/geotiff) by the [Open Geospatial Consortium](https://www.opengeospatial.org/) (OGC), which stands out as a reference when it comes to open geospatial standards.
```

Comment:
Too much information?

Reply:
I don't believe so, it is good to be aware of what the data formats are. Rather, I may even have missed important aspects, perhaps. This is the article, not the tutorials.

Reply:
Agreed, definitely keep it in the article